src/sst/core/interfaces/simpleNetwork.h

```diff
@@ -272,6 +272,29 @@ class SimpleNetwork : public SubComponent {
      */
     virtual Request* recvInitData() = 0;
 
+    /**
+     * Sends a network request during the init() or complete() phase
+     *
+     * For now, simply call sendInitData.  Once that call is
+     * deprecated and removed, this will become a pure virtual
+     * function.
+     */
+    virtual void sendUntimedData(Request *req) {
```

Comment:
Shouldn't this be the other way around?   If `sendInitData()` will be the deprecated route, shouldn't we be moving functionality there, and making the old versions the wrapper?

Reply:
I don't like the way this turned out either, but making the sendInitData the wrapper won't be backward compatible because it is a pure virtual and anything that implements this SubComponent API will be overriding it.  Ideally I would make the new Untimed called pure virtual, but then old code won't compile.  My goal is to allow components using to SimpleNetwork to use the new calls.  However, this will create a problem with implementations of SimpleNetwork in that they will need to either replicate code (implement both init and untimed) or will have to make one call the other.  If you have any ideas how to make this cleaner, I am glad to change how this is implemented.